wordpress-mobile/WordPress-iOS
# Try Gutenberg: Image fullscreen preview

Implements https://github.com/wordpress-mobile/gutenberg-mobile/issues/1286

**PR is for testing a fullscreen preview, does not include gb-mobile release 1.16.0 

Gb-mobile - https://github.com/wordpress-mobile/gutenberg-mobile/pull/1527
Gutenberg - https://github.com/WordPress/gutenberg/pull/17109

To test:

1. Open article in the block editor.
2. Navigate to an image block with an image loaded, or add a new image block and add an image
3. Select the image block and notice the blue border shows around the image
4. Press the image while the image block is selected and notice the image will be viewed in fullscreen.
5. Press outside the boundaries of the image in fullscreen mode, or swipe the image down to exit the fullscreen image preview.

Update release notes:
